stdlib/3/functools.pyi

```diff
@@ -22,10 +22,6 @@ _AnyCallable = Callable[..., Any]
 
 _T = TypeVar("_T")
 _S = TypeVar("_S")
-@overload
```

Comment:
Looks accidental?

Reply:
Oh, this was so that you could see what failure looks like. (Unlike Python) I very much plan on adding reduce back :-)